front/app/containers/Admin/users/UsersGroupHeader.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { memo } from 'react';
+
+import styled from 'styled-components';
+import { Multiloc } from 'typings';
+import Button from 'components/UI/Button';
+import T from 'components/T';
+import Outlet from 'components/Outlet';
+import { MembershipType } from 'services/groups';
+import { FormattedMessage } from 'utils/cl-intl';
+
+import {
+  FirstRow,
+  Spacer,
+  StyledSearchInput,
+  TextAndButtons,
+} from './UsersHeader';
+import messages from './messages';
+
+const OnlyRow = styled(FirstRow)`
```

Comment:
Looking at this, we can use the Box component and pass in these props. It is the preferred approach for new work whenever possible

Reply:
Do you mean I need to change `FirstRow` in `front/app/containers/Admin/users/UsersHeader.tsx`? And `Buttons` in this file too?

Also, I just moved this from another place so maybe it's not worth the effort.

Reply:
Alright. I meant pulling the styles from FirstRow and these ones into a Box 

```
<Box
  display="flex"
  alignItems="flex-start"
  justifyContent="flex-start"
  marginLeft="10px"
  minHeight="105px"
  marginBottom="30px"
>
```

It is minor so I'm happy we can make the changes another time